ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/PaywallViewModel.kt

```diff
@@ -19,14 +20,22 @@ import kotlinx.coroutines.launch
 internal interface PaywallViewModel {
     val state: StateFlow<PaywallViewState>
 
-    fun purchasePackage(activity: Activity, packageToPurchase: Package)
+    fun selectPackage(packageToSelect: Package)
+    fun purchaseSelectedPackage(activity: Activity)
+    fun restorePurchases()
 }
 
-internal class PaywallViewModelImpl(offering: Offering?) : ViewModel(), PaywallViewModel {
+internal class PaywallViewModelImpl(
+    offering: Offering?,
+) : ViewModel(), PaywallViewModel {
+    companion object {
+        const val TAG = "RevenueCatUI"
```

Comment:
Oh looks like this is defined here. Should this be extracted for other classes too?

Reply:
Ah right yeah. We should have it in a single place. And also extract all the logging to a separate place 👍

Reply:
Done in #1233